### PR TITLE
New api gettxout. Update build.gradle to build a jar with dependencies

### PR DESCRIPTION
**Pull request by natzei**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,6 +18,19 @@ jar {
 	}
 }
 
+//create a single Jar with all dependencies
+task fatJar(type: Jar) {
+	manifest {
```
Comment (sebastiandz) [inline build.gradle:23]: Please, add attributes from jar task or at least delete these attribtes.

File: src/main/java/com/sulacosoft/bitcoindconnector4j/response/TransactionOutput.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package com.sulacosoft.bitcoindconnector4j.response;
+
```
Comment (sebastiandz) [inline src/main/java/com/sulacosoft/bitcoindconnector4j/response/TransactionOutput.java]: If you want to add your author signature.

File: src/main/java/com/sulacosoft/bitcoindconnector4j/response/ScriptPubKey.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
+package com.sulacosoft.bitcoindconnector4j.response;
```
Comment (sebastiandz) [inline src/main/java/com/sulacosoft/bitcoindconnector4j/response/ScriptPubKey.java:18]: If you want to add your author signature.

File: src/main/java/com/sulacosoft/bitcoindconnector4j/response/TransactionOutput.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
+package com.sulacosoft.bitcoindconnector4j.response;
```
Comment (sebastiandz) [inline src/main/java/com/sulacosoft/bitcoindconnector4j/response/TransactionOutput.java:18]: If you want to add your author signature.

**Comment by sebastiandz** [comment]
@natzei please look to my coments

**Comment by natzei** [comment]
@sebastiandz amended code

**Pull request merged**